apollo-ios-codegen/Sources/ApolloCodegenLib/Templates/SwiftPackageManagerModuleTemplate.swift

```diff
@@ -23,7 +23,7 @@ struct SwiftPackageManagerModuleTemplate: TemplateRenderer {
     let casedSchemaNamespace = config.schemaNamespace.firstUppercased
 
     return TemplateString("""
-    // swift-tools-version:5.7
+    // swift-tools-version:5.9
```

Comment:
This seems like something we should have updated along the way but I'm not completely sure? How does Xcode react when the generated package requires a lower version than the importing packages (ApolloAPI, etc.) requires?

Reply:
* apollo-ios: updated [8 months ago](https://github.com/apollographql/apollo-ios-dev/blame/main/apollo-ios/Package.swift#L1)
* apollo-ios-codegen: updated [3 months ago](https://github.com/apollographql/apollo-ios-dev/blame/main/apollo-ios-codegen/Package.swift#L1)
* apollo-ios-pagination: updated [3 months ago](https://github.com/apollographql/apollo-ios-dev/blame/main/apollo-ios-pagination/Package.swift#L1)

The template file Blame view shows [June 2023](https://github.com/apollographql/apollo-ios-dev/blame/main/apollo-ios-codegen/Sources/ApolloCodegenLib/Templates/SwiftPackageManagerModuleTemplate.swift#L26) as the last update but that was probably during the project restructuring work. Looking at the version prior to that shows [December 2022](https://github.com/apollographql/apollo-ios/blame/1.5.2/Sources/ApolloCodegenLib/Templates/SwiftPackageManagerModuleTemplate.swift#L19) as the last update, so I feel confident this is something we've simply neglected to update as we do the library version update.